Testscripts/Linux/SR-IOV-Utils.sh

```diff
@@ -76,7 +76,12 @@ VerifyVF()
         vf_interface=$(ls /sys/class/net/ | grep -v 'eth0\|eth1\|lo' | head -1)
     else
         synthetic_interface=$(ip addr | grep "$VF_IP1" | awk '{print $NF}')
-        vf_interface=$(find /sys/devices/* -name "*${synthetic_interface}" | grep "pci" | sed 's/\// /g' | awk '{print $12}')
+        if [[ $DISTRO_VERSION =~ 6\. ]]; then
```

Comment:
could you double check this on CentOS 7.6? On CentOS 7.6, the DISTRO_VERSION is 7.6.1810, which also matches this condition

Reply:
Added a ^ to check whether the DISTRO_VERSION is starting with 6 series